### PR TITLE
 Support for AdditionalFiles

### PR DESCRIPTION
**Pull request by kurobon-jp**
Hello.
I propose this pull request as an answer to [#1510](https://github.com/OmniSharp/omnisharp-roslyn/issues/1510).
github is unfamiliar, but I hope it helps a little.

### CODE REVIEW

**Comment by filipw** [comment]
thanks, could you add a test for this?

**Comment by kurobon-jp** [comment]
Hello.
I added the AdditionalFiles test.

File: src/OmniSharp.MSBuild/ProjectManager.cs [hunk 1]
```diff
@@ -449,6 +450,22 @@ private void UpdateAnalyzerReferences(ProjectFileInfo projectFileInfo, Project p
...
 
+        private void UpdateAdditionalFiles(Project project, IList<string> additionalFiles)
+        {
+            var currentAdditionalDocuments = project.AdditionalDocuments;
+            foreach (var document in currentAdditionalDocuments)
+            {
+                _workspace.RemoveAdditionalDocument(document.Id);
+            }
+
+            foreach (var file in additionalFiles) {
```
Comment (filipw) [inline src/OmniSharp.MSBuild/ProjectManager.cs]: nit: could you move `{` to the new line?

**Comment by kurobon-jp** [comment]
> I think code side looks good, did you test in practise that these changes fixes #1547 ? Based on StyleCopAnalyzers repository documentation it indeed should but didn't test this by myself yet.

Of course, I confirmed that the setting of stylecop.json was reflected.
But that may not be perfect.

**Comment by david-driscoll** [comment]
Thanks for the contribution @kurobon-jp!

**Comment by filipw** [comment]
👏

**Pull request merged**